plugins/woocommerce-admin/client/homescreen/activity-panel/orders/index.js

```diff
@@ -215,58 +224,86 @@ function OrdersPanel( { unreadOrdersCount, orderStatuses } ) {
 			_fields: [
 				'id',
 				'number',
+				'currency',
 				'status',
-				'total_formatted',
+				'total',
 				'customer',
-				'products',
+				'line_items',
 				'customer_id',
 				'date_created_gmt',
 			],
 		} ),
 		[ orderStatuses ]
 	);
-	const { orders = [], isRequesting, isError } = useSelect( ( select ) => {
-		const { getItems, getItemsError, isResolving } = select(
-			ITEMS_STORE_NAME
-		);
 
-		if ( ! orderStatuses.length && unreadOrdersCount === 0 ) {
-			return { isRequesting: false };
+	const currencyContext = useContext( CurrencyContext );
+
+	const getFormattedOrderTotal = ( total, countryState ) => {
+		if ( ! countryState ) {
+			return null;
 		}
 
-		/* eslint-disable @wordpress/no-unused-vars-before-return */
-		const orderItems = getItems( 'orders', actionableOrdersQuery, null );
+		const country = getCountryCode( countryState );
+		const { currencySymbols = {}, localeInfo = {} } = getAdminSetting(
+			'onboarding',
+			{}
+		);
+		const currency = currencyContext.getDataForCountry(
+			country,
+			localeInfo,
+			currencySymbols
+		);
+		currencyContext.setCurrency( currency );
+		return currencyContext.formatAmount( total );
+	};
+
+	const { orders = [], isRequesting, isError, customerItems } = useSelect(
+		( select ) => {
+			const { getOrders, hasFinishedResolution, getOrdersError } = select(
+				ORDERS_STORE_NAME
+			);
+			// eslint-disable-next-line @wordpress/no-unused-vars-before-return
+			const { getItems } = select( ITEMS_STORE_NAME );
+
+			if ( ! orderStatuses.length && unreadOrdersCount === 0 ) {
+				return { isRequesting: false };
+			}
+
+			/* eslint-disable @wordpress/no-unused-vars-before-return */
+			const actionableOrders = getOrders( actionableOrdersQuery, null );
+
+			const isRequestingActionable = hasFinishedResolution( 'getOrders', [
+				actionableOrdersQuery,
+			] );
+
+			if (
+				isRequestingActionable ||
+				unreadOrdersCount === null ||
+				actionableOrders === null
+			) {
+				return {
+					isError: Boolean( getOrdersError( actionableOrdersQuery ) ),
+					isRequesting: true,
+					orderStatuses,
+				};
+			}
 
-		const isRequestingActionable = isResolving( 'getItems', [
-			'orders',
-			actionableOrdersQuery,
-		] );
+			const customers = getItems( 'customers', {
+				users: actionableOrders
+					.map( ( order ) => order.customer_id )
+					.filter( ( id ) => id !== 0 ),
+				_fields: [ 'id', 'name', 'country', 'user_id' ],
+			} );
```

Comment:
customer data is not available through the orders API, so we retrieve them from the Analytics customers API, as we link to the customer analytics page.
I added the `users` param to this API so we can retrieve customers by the user id instead of customer id.